src/main/resources/templates/db_scripts/purchase_order_table.sql

```diff
@@ -2,4 +2,4 @@ CREATE SEQUENCE IF NOT EXISTS ${myuniversity}_${mymodule}.po_number MAXVALUE 999
 GRANT USAGE ON SEQUENCE ${myuniversity}_${mymodule}.po_number TO ${myuniversity}_${mymodule};
 
 
-CREATE UNIQUE INDEX IF NOT EXISTS purchase_order_po_number_unique_idx ON ${myuniversity}_${mymodule}.purchase_order ((jsonb->>'po_number'));
+CREATE UNIQUE INDEX IF NOT EXISTS purchase_order_po_number_unique_idx ON ${myuniversity}_${mymodule}.purchase_order ((jsonb->>'poNumber'));
```

Comment:
I believe this change will work only for new installation but I think will not be picked up on upgrade for any existing environment.

Reply:
I think you're right @piotr-kalashuk.  IIRC @komick, @KVupp and I discussed this briefly last week.  

This PR is already large enough, let's hold off on this and address it afterwards.  I think we might want to do a `DROP INDEX IF EXISTS` and then always create the index.

Reply:
linked to this from [MODORDSTOR-64](https://issues.folio.org/browse/MODORDSTOR-64) so it doesn't get lost